tests/integration/workflows/test_hep_approval.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2017 CERN.
```

Comment:
Nit: wrong year. One day I will figure out where you get these wrong headers... : )

Reply:
Oh, I copy paste them and then fix them :P Thought the idea was to have only the last year ;)